### DISCUSSION TITLE
Final thoughts on #3558 and #3559

### DISCUSSION THREAD

**Discussion by mixonomicon** [reactiveui/ReactiveUI#3560] [General]
@glennawatson @ChrisPulman 

Re #3558 and #3559

I will indeed try to resolve this at System.Reactive level, am actually doing it right now, but I need to produce something before I approach them with the proposed solution. I still do need a temporary resolution so I can get my team rolling on the new wpf library, and I wouldn't have opened another issue if you didn't have the conversation lock on closed issues (honestly one of the rudest thing GitHub allows). I apologize for doing that, but in the absence of DMs here, I have no other way to communicate with you. Other than maybe being persistent, I believe I have not been rude or offensive to deserve being silenced. 

If you do not wish to accept this temporary solution, that is fine, all I wanted was an opportunity to present it. I am not without recourse, for our purposes we'll just build an internal 19.2.2 packages from a fork stripped to WPF only and place them in our own nuget server. **My take was simply that others might be interested in this solution too.**

I will approach you again when and if I manage to reach an accord with System.Reactive team. Hopefully it will gain more traction with you at that time. I wont be bothering you again until then.

Thanks for reading.

**Comment by glennawatson** [comment]
I believe you have been absolutely rude by persisting in occupying the entire teams time. Open source our time is precious to everyone what limited we have. We appreciate having issues raised for sure but we do close them for a reason. 

There is a chat slack server you are welcome to join where you may be able to discuss further if you like on the #reactiveui and #rxnet channels. You can potentially coordinate changes there and get the feel of the land there.

**Reply by glennawatson** [reply]
https://www.reactiveui.net/support
